server/server.js

```diff
@@ -10,6 +10,7 @@ const app = express();
 app.use(cors());
 app.use(express.static(path.join(__dirname, '/../react-client/dist')));
 
+//  GET Requests
 app.get('/reviewData/:productId', (req, res) => {
   const { productId } = req.params;
   Reviews.findOne({ product_id: productId.toUpperCase() }, (err, result) => {
```

Comment:
Something to play around with later is using async await. Since Mongo & express supports this, you could write something like: 

`app.get('/reviewData/:productId', **async** (req, res) => {
  const { productId } = req.params;
  
  try {
    const reviews = **await** Reviews.findOne({ product_id: productId.toUpperCase() };
    res.status(200);
    res.json(reviews);
    res.end();
  } catch {
    res.sendStatus(404);
    res.end();
  }
});`

It's just a different paradigm to try out. It helps me keep my code more linear though & allows me to easily expand any given request.